src/Evolution/Systems/Cce/OptionTags.hpp

```diff
@@ -0,0 +1,103 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "Evolution/Systems/Cce/ReadBoundaryDataH5.hpp"
+#include "NumericalAlgorithms/Interpolation/SpanInterpolator.hpp"
+#include "Options/Options.hpp"
+
+namespace Cce {
+/// \cond
+class Interpolator;
+/// \endcond
+
+namespace OptionTags {
+
+struct LMax {
+  using type = size_t;
+  static constexpr OptionString help{
+      "maximum l value for spin-weighted spherical harmonics"};
+};
+
+struct ObservationLMax {
+  using type = size_t;
+  static constexpr OptionString help{"maximum l value for swsh output"};
+};
+
+struct NumberOfRadialPoints {
+  using type = size_t;
+  static constexpr OptionString help{
+      "Number of radial grid points in the spherical domain"};
+};
+
+struct EndTime {
+  using type = double;
+  static constexpr OptionString help{"End time for the Cce Evolution."};
+  static double default_value() noexcept {
+    return std::numeric_limits<double>::quiet_NaN();
+  }
+};
+
+struct StartTime {
+  using type = double;
+  static constexpr OptionString help{"Start time for the Cce Evolution."};
```

Comment:
Make default some standard value (-inf or quiet NaN), in which case it chooses a "reasonable" start time (and document what that means)